### PR TITLE
Add Zip Pay Icon

### LINKED ISSUE #10174: Add Zip Pay
### Brand Name

Zip Pay

### Website

https://zip.co/au

### Popularity Metric

20.696 - https://www.similarweb.com/website/zip.co/

### Official Resources for Icon and Color

https://developers.zip.co/docs/footer-payment-icons

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by jrhaberland**
zippay

**Issue:** closes #10174

**Popularity metric:**
29,304 - https://www.similarweb.com/website/zip.co/

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
I chose the color of "#1A0826" because that is one of the most prevalent colors on their [media kit](https://zip.co/investors/global-media-kit) page.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -17993,6 +17993,12 @@
             "hex": "009CFB",
             "source": "https://www.zingat.com"
         },
+        {
+            "title": "Zip Pay",
+            "hex": "1A0826",
+            "source": "https://zip.co",
+            "guidelines": "https://zip.co/investors/global-media-kit"
```
Comment (LitoMore) [inline _data/simple-icons.json:18000]: The link is not available anymore.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (jrhaberland) [reply _data/simple-icons.json:18000]: Yup the took the guidelines completely off their site, I can't find the exact same thing. But found something similar, so I'll replace that link with that I found: https://zip.co/nz/marketing/the-logo/

**Comment by uncenter** [comment]
See https://github.com/simple-icons/simple-icons/issues/10174#issuecomment-2350805512. I believe we will need explicit permission for this one.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -17941,6 +17941,12 @@
             "hex": "009CFB",
             "source": "https://www.zingat.com"
         },
+        {
+            "title": "Zip Pay",
```
Comment (uncenter) [inline _data/simple-icons.json:17945]:
```suggestion
            "title": "Zip",
```
The name seems to be just "Zip"? Not seeing "Zip Pay" anywhere but guessing it is one of their products?

File: _data/simple-icons.json [hunk 3]
```diff
@@ -17941,6 +17941,12 @@
...
+            "hex": "1A0826",
+            "source": "https://zip.co",
```
Comment (uncenter) [inline _data/simple-icons.json:17947]: Let's use the icon provided in the press kit linked below as the source, make sure to verify against the current or update the icon.

```suggestion
            "source": "https://zip.co/nz/marketing/the-logo/",
```

File: _data/simple-icons.json [hunk 4]
```diff
@@ -17941,6 +17941,12 @@
             "hex": "009CFB",
             "source": "https://www.zingat.com"
         },
+        {
+            "title": "Zip Pay",
+            "hex": "1A0826",
```
Comment (uncenter) [inline _data/simple-icons.json:17946]:
```suggestion
            "hex": "AA8FFF",
```

> Our brand is rooted in our purple Fearlessness. When we think of Zip we think of this energetic and bold color. Our Fearless hero lives alongside Confidence and Control.

**Comment by uncenter** [comment]
The brand guidelines pdf linked at https://zip.co/us/merchant-resources says we can't (or at least shouldn't) use the logo in non-approved/specified colorways, monochrome does not seem to be one of these colorways.

**Comment by adamrusted** [comment]
As per https://github.com/simple-icons/simple-icons/pull/11179#discussion_r1759653441 I'm gonna close this as we've not reached out to the brand. Feel free to ping here if you do reach out to them @jrhaberland!

**Pull request closed without merging**